### PR TITLE
refactor!: remove deprecated seo option

### PR DESCRIPTION
**Pull request by divine**

### CODE REVIEW

**Comment by divine** [comment]
Hello,

`next` branch is out of sync, how we should proceed @rchl.

Thanks!

File: test/module.test.js [hunk 1]
```diff
@@ -17,8 +17,7 @@ describe('locales as string array', () => {
   let nuxt
 
   beforeAll(async () => {
-    const overrides = { i18n: { seo: false } }
-    const testConfig = loadConfig(__dirname, 'no-lang-switcher', overrides, { merge: true })
+    const testConfig = loadConfig(__dirname, 'no-lang-switcher', { merge: true })
```
Comment (rchl) [inline test/module.test.js]:
```suggestion
    const testConfig = loadConfig(__dirname, 'no-lang-switcher')
```

**Comment by rchl** [comment]
> `next` branch is out of sync, how we should proceed @rchl.

I've rebased `next` on `master` and force-pushed.

**Comment by rchl** [comment]
I'd like to refactor the SEO documentation but this looks good for now. Thanks.

**Review by rchl — APPROVED** [review]

**Pull request merged**